### ISSUE TITLE
No "Skip to main content" button on any pages

### ISSUE THREAD

**Issue by jchristgit** [python-discord/site#1386]
As pointed out in #1363:

There is no mechanism to skip past the navigation bar.

Repetative content across pages in a website, such as a navigation bar, should be easily skippable by keyboard users. Ideally, the first interactable element should be a “Skip to main content” link that goes to the h1 tag, or the equivalent where the main part of the page starts. For reference, check out Python’s Wikipedia page, where pressing tab the first time skips to the wiki title.
